CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased changes
+- Removed deprecated V1 API from the SDK.
```

Comment:
.. and associated types.

Reply:
And also, I think we should add one sentence here mentioning that consumers (of the now removed) `GetBlockSummary` endpoint should refer to `GetBlockTransactionEvents`, `GetBlockSpecialEvents` and `GetBlockPendingUpdates`.